### PR TITLE
Ct one table 2.10

### PR DESCRIPTION
**Pull request by mishuang2017**
Roi, I updated the comment. Actually, I did a clone git from the scratch. So I think I needn't to do rebase. I don't know why there are two commits.

### CODE REVIEW

**Comment by roidayan** [comment]
please update the commit msg for "tc: Fix enum order to keep compatibility"
should mention its fior ct nat . also specify fixes line.

tc: Fix ct nat enum ordering to keep compatibility

So kernels without nat can still work with updated ovs.

**Comment by roidayan** [comment]
also you need to bease the patchset. the fallback commit already merged

File: lib/netdev-tc-offloads.c [hunk 1]
```diff
@@ -1328,6 +1328,13 @@ netdev_tc_flow_put(struct netdev *netdev, struct match *match,
             const struct nlattr *ct_attr;
             size_t ct_left;
 
+            /*
+             * OVS can assembly the fragmented packets and do CT actions,
+             * but TC doesn't support that currently. So return EOPNOTSUPP.
+             */
+            if (flower.key.flags & TCA_FLOWER_KEY_FLAGS_IS_FRAGMENT)
+                return EOPNOTSUPP;
+
```
Comment (roidayan) [inline lib/netdev-tc-offloads.c]: this change already exists so probably need to rebase

**Comment by roidayan** [comment]
this is not rebased. cannot take it.

**Comment by mishuang2017** [comment]
I cloned from your repo and create a pull request again. It seems now there is only one commit.

**Comment by roidayan** [comment]
merged manually.

**Pull request closed without merging**